priam/src/main/java/com/netflix/priam/backupv2/MetaFileWriterBuilder.java

```diff
@@ -65,7 +65,7 @@ public StartStep newBuilder() throws IOException {
     }
 
     public interface DataStep {
-        DataStep addColumnfamilyResult(
+        ColumnfamilyResult addColumnfamilyResult(
```

Comment:
Minor persoanl nit: would rename to `ColumnFamilyResult` (capital F) to keep with Cassandra naming